graphite/graphite.go

```diff
@@ -26,12 +26,30 @@ type Series struct {
 
 type DataPoint []json.Number
 
+type Error struct {
```

Comment:
graphite.Error can conceptually contain error information about anything wrong when doing anything graphite related. it'll probably be mostly/all request related but i wouldn't restrict ourselves if we don't need to.